<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignInDlg</class>
 <widget class="QWidget" name="SignInDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>484</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>484</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>答题器</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#SignInDlg{
	background-color: rgb(211, 211, 211);
}	</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/widget/img/widget/back.png) 6 352 18 0;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget
{
	background-color: rgb(255, 255, 255);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(82, 204, 144);

</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>7</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 16px &quot;微软雅黑&quot;;</string>
           </property>
           <property name="text">
            <string>签到</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClose">
           <property name="minimumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{	
	border-image: url(:/Login/img/Login/close_green_Normal.png);
}

QPushButton:hover{		
	border-image: url(:/Login/img/Login/close_green_Click.png);
}

QPushButton:pressed{
	border-image: url(:/Login/img/Login/close_green_Normal.png);
}

QPussButton:disable{
	border-image: url(:/Login/img/Login/close_green_Normal.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_15">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSignIn">
          <property name="minimumSize">
           <size>
            <width>202</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>202</width>
            <height>34</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:checked{
border-top-left-radius : 15px; 
border-bottom-left-radius : 15px; 
background-color: rgb(82, 204, 144);
font: 14px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
}


QPushButton:!checked{
border-top-left-radius : 15px; 
border-bottom-left-radius : 15px; 
background-color: rgb(255, 255, 255);
font: 14px &quot;微软雅黑&quot;;
color: rgb(68, 68, 68);
border:1px solid rgba(196,196,196,255);
}</string>
          </property>
          <property name="text">
           <string>签到</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSignInRecord">
          <property name="minimumSize">
           <size>
            <width>202</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>202</width>
            <height>34</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:checked{
border-top-right-radius : 15px; 
border-bottom-right-radius : 15px; 
background-color: rgb(82, 204, 144);
font: 14px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
}


QPushButton:!checked{
border-top-right-radius : 15px; 
border-bottom-right-radius : 15px; 
background-color: rgb(255, 255, 255);
font: 14px &quot;微软雅黑&quot;;
color: rgb(68, 68, 68);
border:1px solid rgba(196,196,196,255);
}</string>
          </property>
          <property name="text">
           <string>签到记录</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_16">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true">#stackedWidget
{
/*
	border:1px;
  border-left: 1px solid rgb(196, 196, 196);  
  border-right: 1px solid rgb(196, 196, 196);*/
}
</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_teacher_select">
         <property name="styleSheet">
          <string notr="true">QWidget#page_teacher_select{
background-color: rgb(255, 255, 255);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>79</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnStartSignIn">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
border-top-left-radius : 15px; 
border-top-right-radius : 15px; 
border-bottom-left-radius : 15px; 
border-bottom-right-radius : 15px; 
background-color: rgb(82, 204, 144);
font: 12px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
}

QPushButton:hover{
border-top-left-radius : 15px; 
border-top-right-radius : 15px; 
border-bottom-left-radius : 15px; 
border-bottom-right-radius : 15px; 
background-color: rgb(66, 198, 97);
font: 10pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
}</string>
              </property>
              <property name="text">
               <string>开始签到</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="labCountDown">
            <property name="styleSheet">
             <string notr="true">
color:rgba(153,153,153,1);
font: 14px &quot;微软雅黑&quot;;</string>
            </property>
            <property name="text">
             <string>点击按钮，开始签到</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>79</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labStatistics">
            <property name="styleSheet">
             <string notr="true">color:rgba(68,68,68,255);
font: 14px &quot;微软雅黑&quot;;</string>
            </property>
            <property name="text">
             <string>当前签到人数：&lt;font color = #173f26&gt; 0&lt;/font&gt;人</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_teacher_pushAnswer">
         <property name="styleSheet">
          <string notr="true">QWidget#page_teacher_pushAnswer{
background-color: rgb(255, 255, 255);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>30</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>30</number>
          </property>
          <property name="bottomMargin">
           <number>15</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QWidget#widget_3{
background:rgba(255,247,233,255);
border:1px solid rgba(250,206,116,255);
}
</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <spacer name="horizontalSpacer_7">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_tatal">
                 <property name="minimumSize">
                  <size>
                   <width>95</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(68, 68, 68);
font: 14px &quot;微软雅黑&quot;;</string>
                 </property>
                 <property name="text">
                  <string>在线人数：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="line">
                 <property name="minimumSize">
                  <size>
                   <width>1</width>
                   <height>14</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>1</width>
                   <height>14</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">
background-color: rgb(192, 192, 192);</string>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_sigin">
                 <property name="minimumSize">
                  <size>
                   <width>95</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(68, 68, 68);
font: 14px &quot;微软雅黑&quot;;</string>
                 </property>
                 <property name="text">
                  <string>签到人数：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="line_3">
                 <property name="minimumSize">
                  <size>
                   <width>1</width>
                   <height>14</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>1</width>
                   <height>14</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">
background-color: rgb(192, 192, 192);</string>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_startTimeCount">
                 <property name="minimumSize">
                  <size>
                   <width>95</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(68, 68, 68);
font: 14px &quot;微软雅黑&quot;;</string>
                 </property>
                 <property name="text">
                  <string>签到率：</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_8">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QListWidget
{
	background-color: rgb(255, 255, 255);
	border:0px;
}

/*滚动条样式 开始*/
QScrollBar:vertical
{
border-radius:4px;
width:6px;
background:rgba(255,255,255, 255);
margin:0px,0px,0px,0px;
overflow-x:hidden;
}

QScrollBar::handle:vertical
{
background:rgba(210,210,210, 255);
border-radius:3px;
min-height:20;
border: 1px solid #D2D2D2;
}
</string>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_6">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/widget/img/widget/back.png) 10 0 18 352;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_7">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/widget/img/widget/back.png) 182 0 0 0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>10</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/widget/img/widget/back.png) 0 0 194 0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CWidget</class>
   <extends>QWidget</extends>
   <header>cwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../VLive/VLive.qrc"/>
 </resources>
 <connections/>
</ui>
